projects/admin/src/app/record/detail-view/holding-detail-view/serial-holding-detail-view/expected-issue/expected-issue.component.html

```diff
@@ -0,0 +1,31 @@
+<!--
+  RERO ILS UI
+  Copyright (C) 2022 RERO
+  Copyright (C) 2022 UCLouvain
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, version 3 of the License.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<div class="issue expected-issue container {{ class }}" >
+  <div class="row">
+    <div class="col row">
```

Comment:
seem wired `col` and `row` on the same html tag. https://getbootstrap.com/docs/3.4/css/#grid-nesting